source/basic.tex

```diff
@@ -4060,8 +4059,8 @@
 \tcode{void}~(\ref{basic.types}) has three corresponding cv-qualified
 versions of its type: a \defn{const-qualified} version, a
 \defn{volatile-qualified} version, and a
-\defn{const-volatile-qualified} version. The term
-\term{object type}~(\ref{intro.object}) includes the cv-qualifiers
+\defn{const-volatile-qualified} version. The
+type of an object~(\ref{intro.object}) includes the \grammarterm{cv-qualifier}s
```

Comment:
Use {cv-qualifier}{s} to avoid the \xspace inside \grammarterm inserting a space before the 's'.